README.md

```diff
@@ -1,5 +1,96 @@
-# Umbra
+<div align="center">
+	<img width="400" src="readme/umbra-logo-words.png" alt="Umbra Logo">
+	<br />
+	<br />
+</div>
 
-Send and receive stealth payments.
+<p align="center">
+	<b>Privacy Preserving Shielded Payments On The Ethereum Blockchain.</b>
+</p>
 
-See the README files within the subfolders for instructions on how to use each module.
+<p align="center">
+	🚀 <a href="https://ropsten.umbra.cash">umbra.cash</a>
+	👷 <a href="https://twitter.com/msolomon44">@msolomon44</a>
+	👷 <a href="https://twitter.com/BenDiFrancesco">@BenDiFrancesco</a>
+</p>
+
+<div align="center">
+	<img width="150" src="readme/ethereum-badge-light.png" alt="Umbra Logo">
+	<br />
+</div>
+
+## About
+
+Umbra is protocol for enabling shielded payments on the Ethereum blockchain. It enables privacy preserving transactions where **the receiver's identity is only known to the sender**.
+
+This repository has three components:
+
+* [Umbra JS](umbra-js/) — A JavaScript library for building Umbra-enabled web3 apps in node.js or in the browser.
+* [Umbra Contracts](contracts/) — Solidity contracts used in the Umbra Protocol.
+* [Umbra Pay](app/) — Frontend web3 app for setting up and using Umbra, deployed at [https://ropsten.umbra.cash](https://ropsten.umbra.cash)
+
+### FAQ
+
+**What does Umbra do?**
+
+Umbra allows a payer to send funds to a fresh address. That address is controlled by the intended receiver, but only the payer and the receiver know that.
+
+**How is Umbra different from Tornado Cash**
+
+[Tornado Cash](https://tornado.cash/) is awesome and you should use it! It's an on chain mixer that uses zero knowledge proofs. You put funds in, wait a while for other people to do the same, then use your proof to withdrawal them somewhere else. Since everyone's funds are pooled in the mixer, the link between the origin address and withdrawal address is broken.
+
+Umbra is meant for payments between two entities, and comes with a different set of privacy tradeoffs. Rather than *breaking* the link between sending and receiving address, Umbra makes that link *meaningless*. Everyone know the address funds were sent to, but they don't know who controls that address.
```

Comment:
Huh. Never thought of this, but you're right. It totally could. I'm not sure we should mention it here, because it might confuse the explanation, but I don't see any reason why this wouldn't work.

Reply:
Yea I think it's ok not to mention it here. But definitely an application to keep in mind